python/examples/integrations/writers/Writing_to_WhyLabs.ipynb

```diff
@@ -344,25 +344,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 7,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<Response [200]>"
-      ]
-     },
-     "execution_count": 4,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
```

Comment:
I'm curious why this had an output before but not now.

Reply:
That's actually a change on how the `writer` works: it used to return the call's response, like [200], but it returns `None` now.

I actually feel like it would be nice to have a response to let the user know it went ok. I'll open a discussion for it and maybe open an issue for it.

Reply:
Sounds great